portfolio/src/main/webapp/task.js

```diff
@@ -3,6 +3,8 @@ const TIME_UNIT = {
   HOURS: 'hours'
 };
 
+var ID_COUNTER = 0;
```

Comment:
Consider renaming to TASK_ID_COUNTER for clarity.

Reply:
Done.